macaroon-minting.xml

```diff
@@ -52,7 +52,7 @@
       was issued.</t>
 
       <t>This document describes a simple API that allows a client to
-      request a macaroon be minted that captures the client's
+      request a macaroon to be minted that encodes the client's
```

Comment:
neither do I know, yet it sounded strange to me and with the 'to' it sounds way better to me ...